### PR TITLE
fix sample track out of sync with jack on 48000 hz

### LINKED ISSUE #4786: sample track out of sync with jack
Hi,
my sample tracks are out of sync with the midi tracks when not starting playback from the beginning.

I'm using 1.2.0 rc7 on Linux.

### PR DESCRIPTION
**Pull request by BaraMGB**
fixes  https://github.com/LMMS/lmms/issues/4786

### CODE REVIEW

**Comment by Spekular** [comment]
For whatever my code review is worth:

No cosmetic issues since you're just replacing functions, no concern about the actual logic of the fix either. Have you tested if this fix works with oversampling enabled?

**Review by Spekular — APPROVED** [review]

File: src/core/SampleBuffer.cpp [hunk 1]
```diff
@@ -378,10 +378,10 @@ void SampleBuffer::normalizeSampleRate( const sample_rate_t _src_sr,
 							bool _keep_settings )
 {
 	// do samplerate-conversion to our default-samplerate
```
Comment (Spekular) [inline src/core/SampleBuffer.cpp]:
```suggestion
	// do samplerate-conversion to our processing-samplerate
```

**Review by Spekular — CHANGES_REQUESTED** [review]

File: src/core/SampleBuffer.cpp (hunk 1 above)
Comment (Spekular) [reply src/core/SampleBuffer.cpp]: On second thought, this comment should be changed, unless this line was changed by mistake.

**Comment by BaraMGB** [comment]
I close this in favor of a solution of @Reflexe

**Pull request closed without merging**